vitrina/uapi/urls.py

```diff
@@ -51,4 +50,5 @@
         DistributionViewSet.as_view({"post": "create", "get": "list"}),
         name="uapi-distribution",
     ),
+    path("request-history/<uuid:pk>/", RequestDetailView.as_view(), name="request-history"),
```

Comment:
```suggestion
    path("requests/<uuid:pk>/history", RequestDetailView.as_view(), name="request-history"),
```

Would be a better URL design practice, especially when pk refers to Request.PK (not HistoryEntry.PK)

Reply:
but PK refers to RequestHistory, not to Request.pk and neither to HistoryEntry.PK. Is this a better practice still?

Reply:
It seems more like the path should be smth like:
```
organizations/<int:organization_id>/agents/<uuid:agent_id>/history/<uuid:request_id>/
```

And please move it at the top of the urls, that are template views and not uapi urls

Reply:
moved and changed